notebooks/403-action-recognition-webcam/403-action-recognition-webcam.ipynb

```diff
@@ -0,0 +1,630 @@
+    "\n",
+    "<b id=\"f1\">1</b> seq2seq: Deep learning models that take a sequence of items to the input and output. In this case, input: video frames, output: actions sequence. This `\"seq2seq\"` is composed of an encoder and a decoder. The encoder captures the `\"context\"` of the inputs to be analyzed by the decoder, and finally gets the human action and confidence.[$\\hookleftarrow$](#a1)\n",
+    "\n",
+    "<b id=\"f2\">2</b>  [Video Transformer](https://en.wikipedia.org/wiki/Transformer_(machine_learning_model)#:~:text=A%20transformer%20is%20a%20deep,in%20computer%20vision%20(CV).), and [ResNet34](https://www.kaggle.com/pytorch/resnet34). [$\\hookleftarrow$](#a2)\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "## Imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "import collections\n",
+    "import os\n",
+    "import time\n",
+    "\n",
+    "import cv2\n",
+    "import numpy as np\n",
+    "from IPython import display\n",
+    "from openvino.inference_engine import IECore\n",
+    "from openvino.inference_engine import ExecutableNetwork\n",
+    "\n",
+    "from typing import Optional, Tuple\n",
+    "\n",
+    "sys.path.append(\"../utils\")\n",
+    "import notebook_utils as utils"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## The models\n",
+    "\n",
+    "### Download the models\n",
+    "\n",
+    "We use `omz_downloader`, which is a command-line tool from the `openvino-dev` package. `omz_downloader` automatically creates a directory structure and downloads the selected model.\n",
+    "\n",
+    "In this case you can use `\"action-recognition-0001\"` as a model name, and the system automatically downloads the two models `\"action-recognition-0001-encoder\"` and `\"action-recognition-0001-decoder\"`\n",
+    "\n",
+    "If you want to download another model, such as `\"driver-action-recognition-adas-0002\"` (`\"driver-action-recognition-adas-0002-encoder\"` + `\"driver-action-recognition-adas-0002-decoder\"`), please change the model name. Note: Using a model outside the list can require different pre- and post-processing. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Directory where model will be downloaded\n",
+    "base_model_dir = \"model\"\n",
+    "\n",
+    "# Model name as named in Open Model Zoo\n",
+    "model_name = \"action-recognition-0001\"\n",
+    "\n",
+    "# Selected precision (FP32, FP16, FP16-INT8)\n",
+    "precision = \"FP16-INT8\"\n",
+    "\n",
+    "model_path = f\"model/intel/{model_name}/{model_name}/{precision}/{model_name}.xml\"\n",
```

Comment:
It seems that the model_path is not correct because the check below if the path exists always returns False. There are two models (encoder/decoder), so it is best to check that both exist.

Reply:
I'm not sure if it's related, but I noticed that every time I ran the notebook, the models would download again. It's important that we check and skip when already downloaded. Also, I prefer that we use a cache directory for models similar to what @helena-intel implemented in 104. (and also support moving the models dir and cache dir inside the notebooks folder so it is visible from Jupyter lab web UI)

Reply:
@helena-intel and @adrian: Should I use "model_path" into the `"omz_downloader"`? if the answer is "yes", I could remove the conditional "if" and `"model_weights_path". What do you think?

![image](https://user-images.githubusercontent.com/10940214/149001482-e53ef5fc-9645-4301-bf7a-3d6652246e1a.png)